.github/workflows/test.yml

```diff
@@ -5,12 +5,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - uses: actions/setup-node@v1
+    - uses: borales/actions-yarn@v2.0.0
       with:
-        node-version: 12.x
-    - name: Run npm install, npm test
+        cmd: install
+    - name: Run yarn run test-with-coverage
       run: |
-        npm install
-        npm test
+        yarn run test-with-coverage
       env:
         CI: true
+        CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
```

Comment:
@alexesprit any idea why this is not working? states it cannot find the env var :(

Reply:
It's to push code coverage results to codacy so we can track how many lines are covered with tests etc so we can get a better feel of the tests

Reply:
Does your token work locally? Maybe it's not valid or something else?

Reply:
It works locally for me :/